load_data.py

```diff
@@ -15,13 +15,21 @@
 from data.parse_schema import parse_schema
 from parse_schema import TopicsSchemaParser
 from thresher.models import Article, AnalysisType, TUA, Topic
-
+import pprint;
```

Comment:
Sorry, the java is creeping into my python! This will be removed, I had it there for picking apart the schema json.